lib/iiif_manifest/v3/manifest_builder/annotation_content_builder.rb

```diff
@@ -34,6 +33,18 @@ def generic_annotation
           @generic_annotation ||= iiif_annotation_factory.new
         end
 
+        def annotation_id(canvas_id)
+          if annotation_content.try(:annotation_id).blank?
+            "#{canvas_id}/#{motivation}/#{generic_annotation.index}"
```

Comment:
I think we'll need a fallback here for when `motivation` isn't defined on `annotation_content`.  Maybe something like:
```suggestion
            "#{canvas_id}/#{motivation.blank? ? "annotation" : motivation}/#{generic_annotation.index}"
```